cmd/firearweave/cli/main.go

```diff
@@ -1,90 +0,0 @@
-package cli
```

Comment:
I would like to better understand why this refactoring happened? This is done like this so that a private version of `firehose-arweave` could be created to configure differently the repository.

Reply:
This structure is from https://github.com/streamingfast/firehose-tendermint/tree/main/cmd/firehose-tendermint, `cmd/firehosearweave/cli` is too noisy